### ISSUE TITLE
Problem using "detectobjects" endpoint 

### ISSUE THREAD

**Issue by maxpumperla** [SkymindIO/skil-clients#9]
Working from `skil-python` we're trying to detect objects like this:

```
from skil import Skil, WorkSpace, Experiment, Model, Deployment
import os
import requests
import json
import cv2
import uuid

skil_server = Skil()
work_space = WorkSpace(skil_server)
experiment = Experiment(work_space)
model = Model('yolo_v2.pb', name='yolo', id='yolo-3493723', experiment=experiment)

deployment = Deployment(skil_server, 'yolo')
service = model.deploy(deployment, input_names=['input'], output_names=['output'])

image_path = 'example_bike.jpeg'
file_ = '/home/max/code/yolo-demo/' + image_path
resp = skil_server.api.detectobjects(
    id='foo',
    needs_preprocessing=False,
    threshold='0.5',
    image_file=file_,
    deployment_name=model.deployment.name,
    version_name='default',
    model_name=model.name
)
```

No matter what `file_` is, the server will respond with an error code, one such attached here: https://gist.github.com/maxpumperla/6027bf499f3944a90b483920f0fe0bc0

It's either 
a) a documentation problem, because I don't really understand what "image_file" should be
b) a mismatch of params with skil server. it seems in skil server the parameter is called "file", not "imageFile".
c) hopefully a mistake on my side.

**Comment by saudet** [comment]
Yes, it should be "file", at least that's what appears to be causing the NPE here.

**Comment by maxpumperla** [comment]
it's extremely difficult to keep all of this in sync with skil-server development. @wmeddie said he'd ensure that we'll have both "file" and "imageFile" supported in SKIL 1.2.

We should make sure to discourage people from renaming fields in the public API in some way.

**Comment by wmeddie** [comment]
Yes, we need to find some way of making a test suite that can test every api method.  Jackson also has really bad defaults for doing things in backwards compatible ways.  But if something gets added to swagger it should be set in stone until SKIL 2.0.

The problem here is a bug in the swagger file definition that arose due to confusion.  (While imageFile is the name of the field in the class the implemented API expects "file", other properties have the same name.)  We should update the swagger definition of the field name to file.

**Comment by ShamsUlAzeem** [comment]
@maxpumperla @wmeddie 

I updated the param name: https://github.com/SkymindIO/skil-clients/commit/c369a3df1c7a5b1b7de90ad21820c82a0aa49a90

Make sure you generate the python client before trying it out. I'll put this on PyPi if it's working.

**Comment by maxpumperla** [comment]
closed with #11 , released as v1.2.1 on pypi

**Comment by maxpumperla** [comment]
will reflect this upstream in skil-python in the next iteration
